salt/_modules/metalk8s_package_manager_apt.py

```diff
@@ -0,0 +1,142 @@
+'''
+Describes our custom way to deal with yum packages
+so that we can support downgrade in metalk8s
+'''
+import logging
+# pylint: disable=import-error
+import apt      # type: ignore
+import apt_pkg  # type: ignore
+# pylint: enable=import-error
```

Comment:
Please do like for `salt/_modules/metalk8s_cordon.py` using `HAS_LIBS` because here if you not have apt you will get errors